### ISSUE TITLE
On keyboard only navigation, the Sponsors image Grid is not accessible. As well as the images are overlapping.

### ISSUE THREAD

**Issue by HerinHentry** [mr-bobz/talarts#18]
![image](https://cloud.githubusercontent.com/assets/15111528/10789175/ef09e9ac-7dcf-11e5-9494-57413bae6fc4.png)

Please see screenshot when using Keyboard naviagation on Firefox.

**Comment by HerinHentry** [comment]
More over, the Arrows are read as "Link" by the screen reader. They do not have a alt text / title element.

**Comment by mr-bobz** [comment]
This is not valid anymore since we are using image grid now
